formula-r8-tests/src/main/java/com/instacart/formula/r8/fixtures/AbstractClassInheritanceFormulas.kt

```diff
@@ -0,0 +1,24 @@
+package com.instacart.formula.r8.fixtures
+
+import com.instacart.formula.Action
+import com.instacart.formula.ActionFormula
+
+object AbstractClassInheritanceFormulas {
+
+    class One : ActionFormula<Unit, Int>() {
```

Comment:
Both `One` and `Two` extend `ActionFormula`. In our test, we want to make sure that `IFormula.type()` implementation is not modified by R8.

Reply:
This was the primary issue that I noticed with `fullMode=true` enabled